dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild

```diff
@@ -1,49 +1,49 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="7"
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7,8} )
 PYTHON_REQ_USE="sqlite"
+DISTUTILS_USE_SETUPTOOLS=rdepend
 
 inherit distutils-r1
 
 DESCRIPTION="Synchronize calendars and contacts"
 HOMEPAGE="https://github.com/pimutils/vdirsyncer"
 SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
-LICENSE="MIT"
+LICENSE="BSD"
 KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 SLOT="0"
-IUSE="test"
-RESTRICT="!test? ( test )"
+
+PATCHES=( "${FILESDIR}/click-compat.patch" )
```

Comment:
Please include the version, `PV`, when this patch was added.

`vdirsyncer-0.16.8-click-7-compat.patch` or similar.

Reply:
28fe50a7865c0d14e9e17d7f6975c8f95eb8f15a renames the patch file